src/VirtoCommerce.Xapi.Core/UI/graphiql.cshtml

```diff
@@ -0,0 +1,269 @@
+<!--
+ *  Copyright (c) 2024 GraphQL Contributors
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
+ *  LICENSE file in the root directory of this source tree.
+-->
+<!DOCTYPE html>
+<html>
```

Comment:
This file should be placed in the Web project.
And it should be read from disk, not from the embeded resource.